src/_help/customization-options/operations-navigation.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Operations and navigation
+---
+
+- TOC
+{:toc}
+
+Bump.sh offers several methods for sorting your API operations to provide a smooth and coherent reading structure, based on either the tags or the endpoints of your API definition.
+
+The OpenAPI specification allows for several types of requests:
+
+- Endpoints are typically URLs that point to a specific resource on an API server. In OpenAPI, endpoints are stored under the `paths` resource.
+- Webhooks are requests generated by the API and sent to an external server after a specific event has occurred. In OpenAPI 3.1, they are stored under the `webhooks` resource.
+
+Bump.sh can use its resources to group the display of operations in your documentation. You can also use [tags](https://spec.openapis.org/oas/v3.1.0#tag-object), which are used, among other things, to group operations by resource and should be added to your API definition.
+
+![](/images/help/operations.png)
+
+## Group operations by path (endpoints)
+
+When group operations by path is chosen, Bump.sh deduce endpoint name from related path. First part of the path is extracted to generate the endpoint name, and every operation related to this endpoint name will be grouped together.
```

Comment:
```suggestion
When `Group operations by path` is chosen, Bump.sh deduces endpoint names from related paths. The first part of the path is extracted to generate the endpoint name, and every operation related to this endpoint name will be grouped together.
```

Reply:
Not sure how to best present this:
"Group operations by path" is reader-friendlier
`Group by path` is UI-exact
Still, I recommend using one of those as "When group operations by path is chosen" is somewhat hard to read otherwise, from a syntax perspective.

Reply:
Won't stick to UI perfectly as it should be more user-friendly : it doesn't block the reader to find the option in the dashboard.
Not sure of using code blocks for something else than code blocks (this is a personal introspection)